docs/release-files.md

```diff
@@ -119,10 +122,38 @@ and create PR with the relevant updates to the `download/index.html` and
   - Update `download/index.html` with new links.
   - Create PR to the main repository.
 
+# hs-client
+  Since hsd v5 `hs-client` is part of the `hsd`. Original [hs-client repo][hsclient] is now used to
+publish generated content. `hs-client` version will now be strictly tied to
+the `hsd` version. It is then generated from `hsd` code to release separately on
+`git` and `npm`. Most of the process is done by the introduced helper script
+`scripts/gen-hsclient.js`. It can help you setup `hs-client` that just needs
+publishing on `git` and `npm`. It also gives instructions how to do both.
+  After `hsd` has been released we can also release `hs-client` from the same
+commit/tag, just run: `./scripts/gen-hsclient.js` which will generate `hs-client`
+package with `git` setup in `tmp` directory. You can alternatively pass
+`HS_CLIENT_DIR` env variable for custom place. If generating git failed for some
```

Comment:
I guess HS_CLIENT_DIR won't work now, has to be an argument

Reply:
True, forgot to update this doc. Good catch